games-simulation/simgear/simgear-2018.2.2.recipe

```diff
@@ -0,0 +1,64 @@
+SUMMARY="Simulator Construction Gear"
+DESCRIPTION="SimGear is a set of open-source libraries designed \
+to be used as building blocks for quickly assembling 3D simulations, \
+games, and visualization applications."
+HOMEPAGE="https://www.flightgear.org/"
+COPYRIGHT="Curtis Olsen and others"
+LICENSE="LGPL-2.0 AND GPL-2.0 AND MIT"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/project/flightgear/release-2018.2/simgear-$portVersion.tar.bz2"
+CHECKSUM_SHA256="f61576bc36aae36f350154749df1cee396763604c06b8a71c4b50452d9151ce5"
+SOURCE_DIR="simgear-$portVersion"

```

Comment:
* We like to use SourceForge's redirector service :)
* `SOURCE_DIR` ~_can_~ _should_ be omitted because haikuporter already sets it to a default value which is correct here.
```diff
-SOURCE_URI="https://downloads.sourceforge.net/project/flightgear/release-2018.2/simgear-$portVersion.tar.bz2"
+SOURCE_URI="https://downloads.sourceforge.net/flightgear/simgear-$portVersion.tar.bz2"
 CHECKSUM_SHA256="f61576bc36aae36f350154749df1cee396763604c06b8a71c4b50452d9151ce5"
-SOURCE_DIR="simgear-$portVersion"
```